### PR TITLE
699 homepage offer component

### PR DESCRIPTION
**Pull request by Simon323**
__Description:__

Rename offers component -> homepage offers component

### CODE REVIEW

**Comment by magul** [comment]
@jacol12345 we also offline decide, that we are going to move for-loop from inside of `OffersComponent` into `HomepageComponent` and rename `OffersComponent` into `OfferComponent`, so it will be resposible for rendering one offer.

File: frontend/src/app/homepage-offer/homepage-offer.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <span> Od {{ offer.startedAt ? offer.startedAt : 'teraz' }} </span>
+      &nbsp;
+      <span> do {{ offer.finishedAt ? offer.finishedAt : 'ustalenia' }}</span>
+    </section>
+    <div class="offer-organizer">
+      <span aria-hidden="true" class="offer-glyphicon"></span>
+      <span>Organizator: <a href="{{ djangoRoot }}/organizations/{{ offer.organization.slug }}/{{ offer.organization.id }}">{{offer.organization.name }}</a></span>
+    </div>
+  </div>
+</article>
```
Comment (magul) [inline frontend/src/app/homepage-offer/homepage-offer.component.html]: newline

File: frontend/src/app/home/homepage.component.ts [hunk 2]
```diff
@@ -1,10 +1,26 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 
-import { OffersComponent } from '../offers/offers.component';
+import { OffersService } from 'app/homepage-offer/offers.service';
```
Comment (magul) [inline frontend/src/app/home/homepage.component.ts]: can we stick with relative imports for 1st party imports?

File: frontend/src/app/home/homepage.component.ts [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
+import { Offer } from 'app/homepage-offer/offers.model';
 
 @Component({
   selector: 'volontulo-home',
   templateUrl: './homepage.component.html',
-  styleUrls: ['./homepage.component.css']
+  styleUrls: ['./homepage.component.css'],
```
Comment (magul) [inline frontend/src/app/home/homepage.component.ts]: this css is empty, so:
 * delete the file
 * remove this line

File: frontend/src/app/home/homepage.component.ts [hunk 4]
```diff
@@ -1,10 +1,26 @@
...
+  providers: [OffersService]
 })
-export class HomePageComponent {}
+
+export class HomePageComponent implements OnInit {
+  offers: Array<Offer>;
```
Comment (magul) [inline frontend/src/app/home/homepage.component.ts]: `offers: Array<Offer> = [];`

File: frontend/src/app/home/homepage.component.ts (hunk 3 above)
Comment (Simon323) [reply frontend/src/app/home/homepage.component.ts]: We added some style to this file so it should be state in solution.

File: frontend/src/app/homepage-offer/homepage-offer.component.html (hunk 1 above)
Comment (Simon323) [reply frontend/src/app/homepage-offer/homepage-offer.component.html]: New line was added

**Pull request merged**